custom/src/main/java/com/splunk/opentelemetry/micrometer/SplunkMetricsConfig.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.micrometer;
+
+import io.micrometer.signalfx.SignalFxConfig;
+import io.opentelemetry.instrumentation.api.config.Config;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+import java.time.Duration;
+
+class SplunkMetricsConfig implements SignalFxConfig {
+  @Override
+  public boolean enabled() {
+    return Config.get().getBooleanProperty("splunk.metrics.enabled", true);
```

Comment:
Let's limit the number of global/static calls and create a config field on this class that is passed into the constructor.  Same with `Resource` (below).